smbpasswd-web.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Change Passwords
```

Comment:
Could you add a more descriptive description, like "Web interface to smbpasswd"

Reply:
Definitely valid ;-) I updated the commit.

Reply:
Excelent! Thanks!